irr/src/CGLTFMeshFileLoader.h

```diff
@@ -114,12 +117,15 @@ class CGLTFMeshFileLoader : public IMeshLoader
 
 		std::size_t getPrimitiveCount(const std::size_t meshIdx) const;
 
-		void loadNodes() const;
+		void load();
 
 	private:
 		const tiniergltf::GlTF m_gltf_model;
 		CSkinnedMesh *m_irr_model;
 
+		std::vector<std::function<void()>> m_mesh_loaders;
```

Comment:
this only ever has exactly one element. is the abstraction worth it?

Reply:
In practice this will almost always have exactly one element, but in principle a gltf model can have multiple meshes, in which case it should still work correctly.